src/clang_options_data.zig

```diff
@@ -6895,7 +6895,14 @@ joinpd1("mdouble="),
 joinpd1("mfpmath="),
 joinpd1("mhwmult="),
 joinpd1("mthreads"),
-joinpd1("municode"),
+.{
+    .name = "municode",
+    .syntax = .joined,
+    .zig_equivalent = .unicode,
+    .pd1 = true,
+    .pd2 = false,
+    .psl = false,
+},
```

Comment:
This file is generated by tools/update_clang_options.zig. This patch will be reverted with the next run of the tool; the tool needs a corresponding change.